validphys2/src/validphys/tests/test_pythonmakereplica.py

```diff
@@ -21,9 +22,9 @@
 SINGLE_SYS_DATASETS = [
     {"dataset": "DYE886R"},
     {"dataset": "D0ZRAP", "cfac": ["QCD"]},
-    {"dataset": "ATLAS_SINGLETOP_TCH_R_13TEV", "cfac": ["QCD"]},
-    {"dataset": "CMS_SINGLETOP_TCH_R_8TEV", "cfac": ["QCD"]},
-    {"dataset": "CMS_SINGLETOP_TCH_R_13TEV", "cfac": ["QCD"]},
+    {"dataset": "NMC"},
+    {"dataset": "NMCPD"},
```

Comment:
Could you put another `ATLAS` and a `CMS`? I'm not sure what the logic of the previous one was but better if changes to the tests (which is always dangerous) are as minimal as possible in every sense.